cloudbuild/configs/playground-infrastructure/cloudbuild.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  # This step builds the base image for Beam Playground worker
+  - name: 'gcr.io/cloud-builders/docker'
+    args:
+      - 'build'
+      - '--tag=$_REGION-docker.pkg.dev/$PROJECT_ID/$_ARTIFACT_REGISTRY_REPOSITORY_ID/backend-builder:worker'
+      - '-f'
+      - 'cloudbuild/dockerfiles/Dockerfile-base-image-worker'
+      - '.'
+
+  # This step builds worker container and runs precommit gradle task
+  - name: '$_REGION-docker.pkg.dev/$PROJECT_ID/$_ARTIFACT_REGISTRY_REPOSITORY_ID/backend-builder:worker'
+    entrypoint: "/bin/bash"
+    args: ['-c', './gradlew playground:terraform:InitInfrastructure -Pproject_environment="beta"']
+
+  - name: '$_REGION-docker.pkg.dev/$PROJECT_ID/$_ARTIFACT_REGISTRY_REPOSITORY_ID/backend-builder:worker'
+    entrypoint: "gcloud"
+    args:
+      [
+        'container',
+        'clusters',
+        'get-credentials',
+        'playground-backend',
+        '--region=us-central1'
```

Comment:
is this correct that we use hardcoded region here?

Reply:
Changed to cloudbuild yaml native variables